### PR TITLE
Create Exercises_7.cpp

### PR DESCRIPTION
**Pull request by jsxyhelu**
Execrcises a,b,c of Chapter 7. Functors in OpenCV ,without Exercise d

### CODE REVIEW

**Comment by jsxyhelu** [comment]
in my oppinion,Exercise d may be hard to translate into code

File: Exercises_7_a_b_c.cpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
+
+#include "stdafx.h"
```
Comment (garybradski) [inline Exercises_7_a_b_c.cpp]: stdafx.h is particular to windows (pre-compiled headers for speed), best not to include it for everyone.

File: Exercises_7_a_b_c.cpp [hunk 2]
```diff
@@ -0,0 +1,98 @@
+
```
Comment (garybradski) [inline Exercises_7_a_b_c.cpp]: I like comments at the top of every file that tell what it is:
// Answers to exercise 7(a)(b) and (c)

File: Exercises_7_a_b_c.cpp [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+#include "stdafx.h"
+using namespace cv;
+using namespace std;
+
+
```
Comment (garybradski) [inline Exercises_7_a_b_c.cpp]: In all files, I want the program to say what it is:
void help(const char **argv) {
  cout << "\n\n"
       << "Exercises 7a-c:\n"
       << "Call:\n" << argv[0]
       << endl;
}

File: Exercises_7_a_b_c.cpp [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+int main( int argc, const char** argv )
+{
+	/************************************************************************/
```
Comment (garybradski) [inline Exercises_7_a_b_c.cpp]: See above, add in a call:
help(argv);

**Review by garybradski — CHANGES_REQUESTED** [review]
Made some requested changes. Thanks for doing these. I'm just trying to keep everything consistent (and will do it to all the files). See the comments

File: Exercises_7_a_b_c.cpp (hunk 1 above)
Comment (jsxyhelu) [reply Exercises_7_a_b_c.cpp]: OK，i get it.

File: Exercises_7_a_b_c.cpp (hunk 3 above)
Comment (jsxyhelu) [reply Exercises_7_a_b_c.cpp]: in my opinion,exercise 7 does not need something like argv.
Can I omit this item?
just in exercise 7,of course.

**Review by jsxyhelu — COMMENTED** [review]
Exercises at end of Chapter 7,I add the execise 4.
I solved the first half of the problem on paper, but I didn't know how to pass it up

File: Exercises_7_a_b_c.cpp (hunk 3 above)
Comment (garybradski) [reply Exercises_7_a_b_c.cpp]: That's fine, it's just a standard template that I'm using

Comment (jsxyhelu) [reply Exercises_7_a_b_c.cpp]: ok,what should i do to make  these code  be merged?

**Pull request closed without merging**